cli/cli.py

```diff
@@ -27,18 +27,38 @@
 from cli.studio import scenario_cli
 from cli.ultra import ultra_cli
 from cli.zoo import zoo_cli
-
+from pathlib import Path
 
 @click.group()
 def scl():
+    """
+    The SMARTS command line interface.
+    Use --help with each command for further information.
+    """
     pass
 
+def recursive_help(cmd, file, parent=None):
+    ctx = click.core.Context(cmd, info_name=cmd.name, parent=parent)
+    line = "=" * len(cmd.name)
+    file.write(line + "\n" + cmd.name + "\n" + line + "\n\n")
+    file.write(cmd.get_help(ctx) + "\n\n")
+    commands = getattr(cmd, 'commands', {})
+    for sub in commands.values():
+        recursive_help(sub, file, ctx)
+
+@scl.command(name="document-help",help="Write SCL help information to docs.")
+def document():
+    directory = str(Path(__file__).parent.parent)
+    f = open(directory + "/docs/sim/cli.rst", "w")
```

Comment:
Does this `scl document-help` command work in the package version? In that case I think we should just scrap the document entirely.